src/datastore.js

```diff
@@ -77,19 +77,35 @@ class DataStore extends EventEmitter {
 		return _.find(this._data, filter);
 	}
 
-	update(data) {
+	update(data, emitEvents) {
+		if ( emitEvents === undefined ) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of undefined. (no-undefined)](https://app.codacy.com/app/vjfalk/AwesomeCartJS/pullRequest?prid=2579185)